Library/Formula/nomad.rb

```diff
@@ -18,9 +18,19 @@ class Nomad < Formula
 
   depends_on "go" => :build
 
-  go_resource "github.com/shirou/gopsutil" do
-    url "https://github.com/shirou/gopsutil.git",
-      :revision => "ce433bf86e4d05eeae4276aefc462ec825c2a4f5"
+  go_resource "github.com/ugorji/go" do
+    url "https://github.com/ugorji/go/archive/c062049c1793b01a3cc3fe786108edabbaf7756b.tar.gz"
```

Comment:
Use a git checkout rather than a git commit tarball. It's how we handle `go_resource`s and there's no compelling reason to change.